src/js/datadog.js

```diff
@@ -2,11 +2,11 @@ import { get } from 'underscore';
 import { datadogRum } from '@datadog/browser-rum';
 import { datadogLogs } from '@datadog/browser-logs';
 
-import { datadogConfig as config, versions } from './config';
+import { datadogConfig as config, versions, appConfig as app} from './config';
 
 function initLogs({ isForm }) {
   datadogLogs.init({
-    env: _DEVELOP_ ? 'develop' : 'prod',
```

Comment:
We need to add this data to https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/assets/appconfig.json or local is going to fail.  It also removes the necessity for the`app.hasOwnProperty.env` check below

Reply:
Done.